src/v/config/configuration.cc

```diff
@@ -175,12 +175,7 @@ configuration::configuration()
   , max_version(*this, "max_version")
 
   , use_scheduling_groups(*this, "use_scheduling_groups")
-  , enable_admin_api(
-      *this,
-      "enable_admin_api",
-      "Enable the admin API",
-      base_property::metadata{},
-      true)
+  , enable_admin_api(*this, "enable_admin_api")
```

Comment:
Right.  This is kind of an awkward one: in principle if someone had disabled the admin API, I wouldn't want to force them to reenable it.  However, I also don't want to carry this property forward, because disabling the admin API entirely is a Bad Thing now that we use it for config in general, and if someone actually disables it, a bunch of our docs/rpk commands will be invalidated.

I think very few (if any?) users will have been using this property -  it was primarily needed for unit tests that instantiate multiple redpandas.  For anyone who was using it, this is a "rip of the bandaid" approach where we should include this as part of the broader package of 22.1.1 changes around config+admin api.